src/mrpro/data/KHeader.py

```diff
@@ -255,6 +255,97 @@ def from_ismrmrd(
             ) from None
         return instance
 
+    def to_ismrmrd(self) -> ismrmrdschema.ismrmrdHeader:
+        """Create ISMRMRD header from KHeader.
+
+        All the parameters from the ISMRMRD header of the raw file used to create the kheader are saved in the
+        dictionary kheader['misc']. We first fill the information from this dictionary into the ISMRMRD header and then
+        we update the values based on the attributes of the KHeader.
+
+        Returns
+        -------
+            ISMRMRD header
+        """
+        header = ismrmrdschema.ismrmrdHeader()
+
+        # Experimental conditions
+        exp = ismrmrdschema.experimentalConditionsType()
+        exp.H1resonanceFrequency_Hz = self.lamor_frequency_proton
+        header.experimentalConditions = exp
+
+        # Subject information
+        subj = ismrmrdschema.subjectInformationType()
+        subj.patientName = self.patient_name
+        header.subjectInformation = subj
+
+        # Measurement information
+        meas = ismrmrdschema.measurementInformationType()
+        meas.protocolName = self.protocol_name
+        meas.measurementID = self.measurement_id
+        if self.datetime is not None:
+            meas.seriesTime = str(self.datetime.time())
```

Comment:
can this be None?

Reply:
i thought we set it to file modify data if not set in the header or somehting like that..

Reply:
(need to check)